### PR TITLE
add documentation about constant data removal process.

### PR DESCRIPTION
**Pull request by Karim-Mane**
This PR is aimed at addressing issue #171

### CODE REVIEW

File: vignettes/cleanepi.Rmd [hunk 1]
```diff
@@ -218,6 +218,7 @@ dat %>%
 ```
 
 The `remove_constants()` function returns a dataset where all constant columns, empty rows and columns are removed.
+Note that when the first iteration of constant data removal results in a dataset with empty rows and/or columns and constant columns, this process will be carried on several times until there is no more constant data. Rows and columns that were deleted at any iterations will be reported in the report object.
```
Comment (Bisaloo) [inline vignettes/cleanepi.Rmd:221]: I think your current description is good. I just added suggestions to really drive the point home.

```suggestion
The `remove_constants()` function returns a dataset where all constant columns, empty rows and columns are iteratively removed.
Note that when the first iteration of constant data removal results in a dataset with new empty rows and/or columns and constant columns, this process will be carried on several times until there is no more constant data. Rows and columns that were deleted at any iterations will be reported in the report object.
```

**Review by Bisaloo — APPROVED** [review]

**Review by avallecam — APPROVED** [review]
Thanks for adding this. Just wanted to read the report and noticed the issue reported in #192 , but this possibly aims to be solved in a separate PR.

File: R/remove_constants.R [hunk 2]
```diff
@@ -18,23 +18,44 @@
...
 #'   data = data,
 #'   cutoff = 1
 #' )
 #'
-#' # remove the constant columns, empty rows and columns where empty rows and
-#' # columns are defined as those with 50% NA.
-#' dat <- remove_constants(
-#'   data = data,
+#' # drop rows or columns with a percent of constant values equal to or more
+#' # than 50%
```
Comment (avallecam) [inline R/remove_constants.R:37]:
```suggestion
#' # drop rows or columns with a percentage of constant values 
#' # equal to or more than 50%
```

File: R/remove_constants.R [hunk 3]
```diff
@@ -18,23 +18,44 @@
...
+#' test <- cleanepi::remove_constants(
+#'   data = test,
 #'   cutoff = 0.5
 #' )
 #'
+#' # drop rows or columns with a percent of constant values equal to or more
+#' # than 25%
```
Comment (avallecam) [inline R/remove_constants.R:44]:
```suggestion
#' # drop rows or columns with a percentage of constant values
#' # equal to or more than 25%
```

File: R/remove_constants.R [hunk 4]
```diff
@@ -18,23 +18,44 @@
...
+#' test <- cleanepi::remove_constants(
+#'   data = test,
+#'   cutoff = 0.25
+#' )
+#'
+#' # drop rows or columns with a percent of constant values equal to or more
+#' # than 15%
```
Comment (avallecam) [inline R/remove_constants.R:51]:
```suggestion
#' # drop rows or columns with a percentage of constant values
#' # equal to or more than 15%
```

**Review by avallecam — APPROVED** [review]
thanks for adding the examples to show how different `cutoff` values affect an outcome! I suggested three minor edit suggestions for the readability of the sentence by trying to follow semantic line break suggestions like isolating two sections of one sentence. https://sembr.org/

**Comment by Karim-Mane** [comment]
> Thanks for adding this. Just wanted to read the report and noticed the issue reported in #192 , but this possibly aims to be solved in a separate PR.

@avallecam - Thank you for spotting this out. I am fixing this in a separate PR.

**Pull request merged**